sdk/core/src/services/analytics/sboms/service.rs

```diff
@@ -114,6 +115,27 @@ fn report_analytic_stage_6() -> Stage {
     Stage::new(json!({
         "$group": {
           "_id": "$_id",
+          "name": {
+            "$first": "$name"
+          },
+          "packageManager": {
+            "$first": "$packageManager"
+          },
+          "purl": {
+            "$first": "$purl"
+          },
+          "provider": {
+            "$first": "$provider"
+          },
+          "version": {
+            "$first": "$version"
+          },
+          "timestamp": {
```

Comment:
Should we just remove timestamp entirely from the pipeline if it's not used?

Reply:
I actually did that already